substreams-head-tracker/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
Would we have a way to use the top-level one directly. Only thing I'm wondering is the `wasm32-unknown-unknown` target.